turbinia/processors/partitions.py

```diff
@@ -40,12 +39,17 @@ def Enumerate(evidence):
   """
   dfvfs_definitions.PREFERRED_GPT_BACK_END = (
       dfvfs_definitions.TYPE_INDICATOR_GPT)
-  mediator = UnattendedVolumeScannerMediator()
-  mediator.credentials = evidence.credentials
+  options = volume_scanner.VolumeScannerOptions()
+  options.partitions = ['all']
+  options.volumes = ['all']
+  # Not processing volume snapshots
+  options.snapshots = ['none']
+  options.credentials = evidence.credentials
+
   path_specs = []
   try:
-    scanner = volume_scanner.VolumeScanner(mediator=mediator)
-    path_specs = scanner.GetBasePathSpecs(evidence.local_path)
+    scanner = volume_scanner.VolumeScanner(mediator=None)
```

Comment:
Maybe add a comment that setting mediator to None will cause volume scanner to operate in unattended mode.